### DISCUSSION TITLE
Windows Enterprise LTSC support

### DISCUSSION THREAD

**Discussion by ghost** [sandboxie-plus/Sandboxie#2765] [Q&A Feedback] [answered]
Are Windows Enterprise 2019 and 2021 LTSC supported by Sandboxie?

**Comment by DavidXanatos** [comment]
should be, try it out

**Reply by ghost** [reply]
MS will provide support for bugfixes and security patches for Win10 22H1 LTSC 2021 IOT for next 9 years. I wonder if you do too

**Reply by DavidXanatos** [reply]
well sbie is still compatible with 7, for which MSFT does nto provide anythign anymoe so I think we can do better then MSFT and and did so already in the past.
We will keep windows 7 support untill sbie 6.xx

**Comment by ghost** [comment, marked as answer]
I checked with 2021 LTSC IOT and I was able to install SBIE 1.8.1, run notepad in yellow and Edge in orange box
